keepassxc-browser/options/options.html

```diff
@@ -2,644 +2,767 @@
+                </div>
+              </div>
 
-      <!-- Connected Databases -->
-      <div class="tab" id="tab-connected-databases">
-        <h2 data-i18n="optionsConnectedDatabasesTab"></h2>
-        <hr />
-        <table class="table table-striped table-bordered table-hover">
-          <caption data-i18n="optionsConnectedDatabasesText"></caption>
-          <thead>
-            <tr>
-              <th scope="col"><span data-i18n="optionsDatabaseIdentifier"></span></th>
-              <th scope="col"><span data-i18n="optionsDatabaseKey"></span></th>
-              <th scope="col"><span data-i18n="optionsDatabaseLastUsed"></span></th>
-              <th scope="col"><span data-i18n="optionsDatabaseCreated"></span></th>
-              <th scope="col"><span data-i18n="optionsDatabaseDelete"></span></th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr class="empty">
-              <td colspan="5"><span data-i18n="optionsConnectedDatabasesNotFound"></span></td>
-            </tr>
-            <tr class="clone">
-              <td></td>
-              <td></td>
-              <td class="last-used"></td>
-              <td class="created"></td>
-              <td><button class="btn delete btn-danger"><span class="glyphicon glyphicon-remove-sign"></span> <span data-i18n="optionsButtonRemove"></span></button></td>
-            </tr>
-          </tbody>
-        </table>
-        <div style="text-align: right">
-          <button id="connect-button" class="btn btn-primary"><span class="glyphicon glyphicon-link"></span> <span data-i18n="optionsButtonConnect"></span></button>
-        </div>
-
-        <div id="dialogDeleteConnectedDatabase" class="modal fade" tabindex="-1" role="dialog">
-          <div class="modal-dialog">
-            <div class="modal-content">
-              <div class="modal-header">
-                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                <h3 class="modal-title" id="myModalLabel" data-i18n="optionsDatabasesRemoveIdentifier"></h3>
+              <div class="card shadow">
+                <div class="card-header">
+                  <i class="fa fa-keyboard-o" aria-hidden="true"></i>
+                  <span data-i18n="optionsKeyboardShortcutsHeader"></span>
+                </div>
+                <div class="card-body">
+                  <p>
+                    <span data-i18n="contextMenuFillUsernameAndPassword"></span>: <span id="fill_username_password-shortcut">error</span><br />
+                    <span data-i18n="contextMenuFillPassword"></span>: <span id="fill_password-shortcut">error</span><br />
+                    <span data-i18n="contextMenuFillTOTP"></span>: <span id="fill_totp-shortcut">error</span><br />
+                    <span data-i18n="contextMenuShowPasswordGenerator"></span>: <span id="show_password_generator-shortcut">error</span>
+                  </p>
+                  <p id="chrome-only">
```

Comment:
like `chrome-only` could be reused and `id` is only allowed once per page. Most times a `class` is used.